TestCases/turbomachinery/Aachen_turbine/aachen_3D_MP.cfg

```diff
@@ -0,0 +1,363 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: AACHEN turbine 3D                                          %
+% Author: S. Vitale, A. Cappiello                                              %
+% Institution: Delft University of Technology                                  %
+% Date: Oct 20th, 2023                                                         %
+% File Version 8.0 develop                                        %
```

Comment:
```suggestion
```

Reply:
Hi Pedro, wouldn't it be better to keep for future reference the su2 version for which the test case was prepare?

Reply:
The case is now part of the regressions and thus will be kept up to date with the current SU2 version. So it's better to not have something that may become outdated and give wrong information.

Reply:
Alright. I've just removed that line.